opencraft/settings.py

```diff
@@ -300,6 +300,16 @@
 # configuration, edx-platform, forum, notifier, xqueue, and certs when creating production instances.
 OPENEDX_RELEASE_STABLE_REF = env('OPENEDX_RELEASE_STABLE_REF', default='named-release/dogwood.rc')
 
+# The edx-platform repository used by default for production instances
+STABLE_EDX_PLATFORM_REPO_URL = env(
```

Comment:
NIT: This is a nit, so no need to address it, just pointing a thing that we might clean-up: 

 ``STABLE_EDX_PLATFORM_REPO_URL`` is a setting by itself, while ``DEFAULT_EDX_PLATFORM_REPO_URL`` is a constant in instance module, generated from ``DEFAULT_FORK``. 

This is a thing that should be symmetric IMO, either introduce ``DEFAULT_EDX_PLATFORM_REPO_URL`` as a setting, or add ``STABLE_FORK`` setting.